lib/core/jbackend.c

```diff
@@ -106,33 +111,43 @@ j_backend_load (gchar const* name, JBackendComponent component, JBackendType typ
 
 	if (type == J_BACKEND_TYPE_OBJECT)
 	{
-		if (tmp_backend->object.backend_init == NULL
-		    || tmp_backend->object.backend_fini == NULL
-		    || tmp_backend->object.backend_create == NULL
-		    || tmp_backend->object.backend_delete == NULL
-		    || tmp_backend->object.backend_open == NULL
-		    || tmp_backend->object.backend_close == NULL
-		    || tmp_backend->object.backend_status == NULL
-		    || tmp_backend->object.backend_sync == NULL
-		    || tmp_backend->object.backend_read == NULL
-		    || tmp_backend->object.backend_write == NULL)
+		if (tmp_backend->object.backend_init == NULL //
+			|| tmp_backend->object.backend_fini == NULL //
+			|| tmp_backend->object.backend_create == NULL //
+			|| tmp_backend->object.backend_delete == NULL //
+			|| tmp_backend->object.backend_open == NULL //
+			|| tmp_backend->object.backend_close == NULL //
+			|| tmp_backend->object.backend_status == NULL //
+			|| tmp_backend->object.backend_sync == NULL //
+			|| tmp_backend->object.backend_read == NULL //
+			|| tmp_backend->object.backend_write == NULL)
 		{
 			goto error;
 		}
 	}
 
 	if (type == J_BACKEND_TYPE_KV)
 	{
-		if (tmp_backend->kv.backend_init == NULL
-		    || tmp_backend->kv.backend_fini == NULL
-		    || tmp_backend->kv.backend_batch_start == NULL
-		    || tmp_backend->kv.backend_batch_execute == NULL
-		    || tmp_backend->kv.backend_put == NULL
-		    || tmp_backend->kv.backend_delete == NULL
-		    || tmp_backend->kv.backend_get == NULL
-		    || tmp_backend->kv.backend_get_all == NULL
-		    || tmp_backend->kv.backend_get_by_prefix == NULL
-		    || tmp_backend->kv.backend_iterate == NULL)
+		if (tmp_backend->kv.backend_init == NULL //
+			|| tmp_backend->kv.backend_fini == NULL //
+			|| tmp_backend->kv.backend_batch_start == NULL //
+			|| tmp_backend->kv.backend_batch_execute == NULL //
+			|| tmp_backend->kv.backend_put == NULL //
+			|| tmp_backend->kv.backend_delete == NULL //
+			|| tmp_backend->kv.backend_get == NULL //
+			|| tmp_backend->kv.backend_get_all == NULL //
+			|| tmp_backend->kv.backend_get_by_prefix == NULL //
+			|| tmp_backend->kv.backend_iterate == NULL)
+		{
+			goto error;
+		}
+	}
```

Comment:
This contains a lot of formatting changes.

Reply:
As mentioned before, I'll remove those space-only changes